src/components/utils/rhf-inputs/error-inputs/error-input.js

```diff
@@ -16,6 +16,10 @@ const ErrorInput = ({ name, InputField }) => {
         name,
     });
 
+    const { isSubmitted } = useFormState();
```

Comment:
No need this, add 
`formState: { isSubmitted },`
inside the `useController`

Reply:
And why using `isSubmitted` instad of `isSubmitting` ?
Then it focus everytime you submit

Reply:
if you use isSubmitting you risk to trigger the effect before the validation is done, i added error to the useEffect dependencies so it scroll each time you submit